Code/GraphMol/SmilesParse/cxsmiles_test.cpp

```diff
@@ -1468,3 +1468,13 @@ TEST_CASE("write attachment points") {
   m->getAtomWithIdx(5)->setProp(common_properties::_fromAttachPoint, 2);
   CHECK(MolToCXSmiles(*m) == "*N[C@@H](C)C(*)=O |$_AP1;;;;;_AP2;$|");
 }
+
+TEST_CASE("github #7414: CXSmiles writer does not use default conformer ID") {
+  SECTION("basics") {
+    auto m = "CC |(-0.75,0,;0.75,0,)|"_smiles;
+    REQUIRE(m);
+    CHECK(m->getNumConformers() == 1);
+    m->getConformer().setId(5);
+    CHECK(MolToCXSmiles(*m) == "CC |(-0.75,0,;0.75,0,)|");
+  }
+}
```

Comment:
Don't you need two conformers to test this?  From an unpatched rdkit:

```
>>> m = Chem.MolFromSmiles("CC |(-0.75,0,;0.75,0,)|")
>>> m.GetConformer().SetId(5)
>>> Chem.MolToCXSmiles(m)
'CC |(-0.75,0,;0.75,0,)|'
```

Reply:
No, you only need one, but for some reason this molecule isn't triggering the original problem:

```
m = Chem.MolFromSmiles("CC |(-0.75,0,;0.75,0,)|")
m.GetConformer().SetId(5)
m.GetConformer(0)
```

```
ValueError: Bad Conformer Id
```

Reply:
Looks like this bug was introduced after 2023.9.6

Reply:
> Looks like this bug was introduced after 2023.9.6

yeah, it came in with the atropisomer code